flycheck.el

```diff
@@ -6297,6 +6298,27 @@ See URL `https://github.com/w3c/tidy-html5'."
             " - Warning: " (message) line-end))
   :modes (html-mode nxhtml-mode))
 
+(flycheck-def-config-file-var flycheck-jscsrc javascript-jscs ".jscsrc"
+  :safe #'stringp
+  :package-version '(flycheck . "0.24"))
+
+(flycheck-define-checker javascript-jscs
+  "A code style linter using JSCS.
+
+See URL `http://www.jscs.info'."
+  :command ("jscs" "--reporter=checkstyle"
+            (config-file "--config" flycheck-jscsrc)
+            source)
+  :error-parser flycheck-parse-checkstyle
+  :error-filter flycheck-dequalify-error-ids
+  :modes (js-mode js2-mode js3-mode)
+  :predicate
+  ;; JSCS fails with exit code 1 if it cannot find a .jscsrc file.
+  ;; This :predicate prevents such cases.
+  (lambda ()
+    (flycheck-locate-config-file flycheck-jscsrc 'javascript-jscs))
+  :next-checkers (javascript-jshint javascript-eslint javascript-gjslint))
```

Comment:
Style linting should come last.  The other javascript checkers should have this one in their `:next-checkers`.